test-cli/README.md

```diff
@@ -6,7 +6,7 @@ A test cli for the decentralized quoting service.
 
 1. Start the server with:
 
-    MC_LOG=trace IAS_MODE=DEV SGX_MODE=SW cargo run --bin deqs-server -- --client-listen-uri insecure-deqs://127.0.0.1/ --db-path {sqllite_path} --ledger-db {ledgerdb_path}
+    MC_LOG=trace IAS_MODE=DEV SGX_MODE=SW cargo run --bin deqs-server -- --client-listen-uri insecure-deqs://127.0.0.1/ --db-path {sqllite_path} --ledger-db {ledgerdb_path} --quote-minimum-map 0=1000,1=200
```

Comment:
Was this tested?

This results in the example not working:
```
023-03-10 20:10:39.258216 UTC INFO Generating 3 SCIs..., mc.app: deqs-test-cli, mc.module: deqs_test_cli, mc.src: test-cli/src/bin/main.rs:106
2023-03-10 20:10:39.618077 UTC INFO Submitting 3 SCIs..., mc.app: deqs-test-cli, mc.module: deqs_test_cli, mc.src: test-cli/src/bin/main.rs:140
status_codes: UNSUPPORTED_SCI
status_codes: UNSUPPORTED_SCI
status_codes: UNSUPPORTED_SCI
error_messages: "Unsupported SCI: Quote is too small for deqs"
error_messages: "Unsupported SCI: Quote is too small for deqs"
error_messages: "Unsupported SCI: Quote is too small for deqs"
quotes {
}
quotes {
}
quotes {
}


0. UNSUPPORTED_SCI
1. UNSUPPORTED_SCI
```


(I don't think this change is needed, no reason for the server to have minimums when used with the test-cli)

Reply:
Yes, I realize in retrospect that testing that the quotes end up being rejected is only useful for me testing that this change works, and not for anyone else actually trying to use the testcli. :laughing:

Reply:
Makes sense, it's always good to test things locally!